src/Generator.cpp

```diff
@@ -132,6 +132,10 @@ Outputs compute_outputs(const Target &target,
     if (options.emit_schedule) {
         output_files.schedule_name = base_path + get_extension(".schedule", options);
     }
+    if (options.emit_featurization) {
+        // TODO: apparently we use '.sample' for featurization files?
```

Comment:
A sample is a featurization + a runtime + some ids, all together in one file. benchmark_sample above concats the runtime and ids.

Reply:
Mm, so it would perhaps be clearer to have the 'featurization' output to have a different suffix (e.g. `.featurization`), and to have `augment_sample` use that to produce a `.sample` file?

Reply:
Update: pushed a commit that distinguishes between featurization and sample files; PTAL